ui/src/views/404/index.jsx

```diff
@@ -1,72 +1,86 @@
-// @flow
+// @follow
 import Logo from 'assets/images/guardian-logo-320.svg';
+import ArrowRight from 'assets/images/arrow-right.svg';
+import FourOhFour from 'assets/images/404.svg';
```

Comment:
As much as `FourOhFour` made me smile, maybe `NotFound` or `NotFoundSvg` might be better?